Sources/CryptoSwift/LinuxFoundation.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  LinuxFoundation.swift
+//  CryptoSwift
+//
+//  Created by Alsey Coleman Miller on 4/18/16.
+//  Copyright © 2016 Marcin Krzyzanowski. All rights reserved.
+//
+
+import Foundation
+
+#if !os(Linux)
+    
+extension String {
+    
+    // Workaround:
+    // https://github.com/krzyzanowskim/CryptoSwift/issues/177
+    @inline(__always)
+    func bridge() -> NSString {
+        return self as NSString
+    }
+}
+
+/// Fixes discrepancies with Swift 3.0 Linux
```

Comment:
The open source version of Foundation is having problems importing into Swift 3.0, this is only a temporary work around. The compiler said that the base64 methods did not exist on Linux.